Cargo.toml

```diff
@@ -1,5 +1,84 @@
-[workspace]
-members = [
-  "iota-client",
-  "examples",
+[package]
+name = "iota-client"
+version = "1.1.0"
+authors = [
+  "Thoralf Müller <thoralf.mueller@iota.org",
+  "Antony Dikov <antoni.dikov@iota.org>",
+  "Bing-Yang Lin <bingyang.lin@iota.org>",
+  "Lucas Nogueira <lucas.nogueira@iota.org>",
+  "Nathan Jaremko <nathan@jaremko.ca>",
+  "Yu-Wei Wu <wusyong9104@gmail.com>"
 ]
+edition = "2018"
```

Comment:
Maybe we should also update to 2021, wdyt?

Reply:
_Do it! Just do it!_